modules/spring-config/src/main/resources/META-INF/mule-module.properties

```diff
@@ -1,6 +1,7 @@
 module.name=spring-config
 
-artifact.export.classPackages=org.mule.runtime.config.spring.artifact,\
+artifact.export.classPackages=org.mule.runtime.config.spring.api.dsl.processor,\
```

Comment:
The fix should be to avoid exposing the classes... exposing them for everyone instead of just to privileged plugins is not better than the current approach, don't you think?